free-programming-books.md

```diff
@@ -2740,9 +2740,11 @@ Kerridge (PDF) (email address *requested*, not required)
 ### Vim
 
 * [A Byte of Vim](http://www.swaroopch.com/notes/vim/)
+* [Learn Vim (the Smart Way)](https://github.com/iggredible/Learn-Vim) (HTML)
```

Comment:
says it's "in progress". We have a notation for that!

Reply:
@eshellman the book is very well in progress and its evolving but even then, the content there is right now is a lot more helpful than many of the other resources. But, if you think this still can not make it to the list, I am happy to remove it. let me know what you think.

Reply:
add it with the "in progress" notation. please read /CONTRIBUTING.md !!!

Reply:
oops sorry updated now accordingly